ThaumatoAnakalyptor/pointcloud_to_instances.py

```diff
@@ -187,6 +183,13 @@ def post_process_surfaces(surfaces, surfaces_normals, surfaces_colors, scores, s
     surfaces_normals = [surfaces_normals[i] for i in indices]
     surfaces_colors = [surfaces_colors[i] for i in indices]
     scores = [scores[i] for i in indices]
+    
+     # Get the Glasbey colormap
+    glasbey_cmap = cc.cm.glasbey
+    num_colors = len(indices)
+    for i in range(len(indices)):
+        color = glasbey_cmap(i/num_colors)[:3]
+        surfaces_colors[i][:] = color
```

Comment:
Thaumato uses the color attributes to store a uniform random vector to consistently subsample pointclouds across differently cut subvolumes